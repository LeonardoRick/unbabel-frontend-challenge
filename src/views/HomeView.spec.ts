import HomeViewVue from '@/views/HomeView.vue';
import { mount } from '@vue/test-utils';
import { setActivePinia, createPinia } from 'pinia';

describe('HomeView', () => {
    beforeEach(() => {
        setActivePinia(createPinia());
    });

    it('should mount', () => {
        expect(mount(HomeViewVue).html()).toMatchSnapshot();
    });
});
