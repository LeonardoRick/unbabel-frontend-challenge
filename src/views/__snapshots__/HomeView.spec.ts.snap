// Vitest Snapshot v1

exports[`HomeView > should mount 1`] = `
"<div class=\\"wrapper\\">
  <ul></ul><button type=\\"button\\" class=\\"reset-button-style\\"><img src=\\"/src/assets/images/add-row.svg\\" alt=\\"add another transcription\\"></button>
</div>"
`;
